demo/site/src/layout/header/Header.tsx

```diff
@@ -14,14 +15,18 @@ interface Props {
 }
 
 function Header({ header }: Props): JSX.Element {
+    const intl = useIntl();
     return (
         <header>
             <PageLayout grid>
                 <PageLayoutContent>
                     <Root>
                         <nav>
                             <Link href="/">
-                                <SvgUse href="/assets/comet-logo.svg#root" />
+                                <SvgUse
+                                    href="/assets/comet-logo.svg#root"
+                                    title={intl.formatMessage({ id: "header.logo", defaultMessage: "Comet DXP Logo" })}
```

Comment:
What do you think about including title in the ID to demonstrate what the message is used for?

```suggestion
                                    title={intl.formatMessage({ id: "header.logo.title", defaultMessage: "Comet DXP Logo" })}
```

Reply:
The Link above must have a title, regarding the a11y guidelines. Is it then still necessary to let the screen reader know there is a svg (which i can not display) inside the link?

Reply:
According to the guidelines, the Link is not required to have a title - only if its content is not descriptive. SVGs that have a title and are not set to aria-hidden, should be accessible for modern assistive technologies. So I guess, both variants work. 
I still changed the code, as some sources say, that older assistive technologies may not be able to read the SVG correctly.